Makefile

```diff
@@ -31,8 +31,8 @@ battletest: ## Run stronger tests
 	gocyclo -over 11 ./pkg
 	# Run randomized, parallelized, racing, code coveraged, tests
 	ginkgo -r \
-		-cover -coverprofile=coverage.out -outputdir=. -coverpkg=./pkg/... \
-		--randomizeAllSpecs --randomizeSuites -race
+		--cover --coverprofile=coverage.out --output-dir=.\
+		--randomize-all --randomize-suites --race --trace
```

Comment:
Try adding `-coverpkg=github.com/aws/karpenter/pkg/...` , not sure why the -coverpkg=./pkg/... isn't working correctly.